model/riscv_extensions.sail

```diff
@@ -6,6 +6,17 @@
 /*  SPDX-License-Identifier: BSD-2-Clause                                                */
 /*=======================================================================================*/
 
+// Privileged and unprivileged spec versions. The versioning system for specifications
+// is unclear so we just use the commit date. This also allows versioning for specs changes
+// that are between releases. If there was an official release for this day (for example
+// 20240411) then this is the commit of that release. Otherwise it is for the *last*
+// commit on the given day.
+type SpecVersion = range(20000000, 21000000)
+
+// TODO: Replace with 'config ...priv_version' or similar when the config system is available.
+let configPrivVersion : SpecVersion = 20240411
```

Comment:
Maybe the easiest approach here to allow using relational operators would be to add variables such as `Priv_1_12 = 20211203` and use those in the comparisons?

Reply:
And a comment above with a commit hash and a link to the release on github?